packages/dashboard/src/components/lists/List.js

```diff
@@ -19,100 +19,91 @@ import {
   CardBody,
 } from 'shards-react'
 
-
-
-
-const List = ({
-  accounts,
-  user,
-  historicQuotes,
-  globalSelectedAccount,
-}) => {
-  
-
-  
+const List = ({ items, listTitle }) => {
+  const renderListItems = () => {
+    return items.map(item => (
```

Comment:
Same here, map returns an array so forgoing the curly brackets and return statement is a little bit cleaner. Just for future use this is fine now.
```
const renderListItems = () => 
    items.map(item => ....
```

Reply:
I'm not sure if im doing it wrong but it doesn't seem to work properly when I remove the return keyword in the examples you suggested